src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -12,18 +12,19 @@ import org.broadinstitute.dsde.workbench.leonardo.auth.sam.MockPetClusterService
 import org.broadinstitute.dsde.workbench.leonardo.config.{AutoFreezeConfig, ClusterDefaultsConfig, ClusterFilesConfig, ClusterResourcesConfig, DataprocConfig, MonitorConfig, ProxyConfig, SwaggerConfig}
 import org.broadinstitute.dsde.workbench.leonardo.dao.google.MockGoogleComputeDAO
 import org.broadinstitute.dsde.workbench.leonardo.dao.{MockJupyterDAO, MockSamDAO}
+import org.broadinstitute.dsde.workbench.leonardo.db.TestComponent
 import org.broadinstitute.dsde.workbench.leonardo.model._
 import org.broadinstitute.dsde.workbench.leonardo.model.google._
 import org.broadinstitute.dsde.workbench.model.google.{GoogleProject, ServiceAccountKey, ServiceAccountKeyId, ServiceAccountPrivateKeyData, _}
 import org.broadinstitute.dsde.workbench.model.{UserInfo, WorkbenchEmail, WorkbenchUserId}
 import org.scalatest.concurrent.ScalaFutures
 
 import scala.concurrent.ExecutionContext
-import scala.concurrent.duration._
+
 
 // values common to multiple tests, to reduce boilerplate
 
-trait CommonTestData { this: ScalaFutures =>
+trait CommonTestData extends TestComponent with GcsPathUtils{ this: ScalaFutures =>
```

Comment:
I am not sure `CommonTestData` should depend on `TestComponent`. I think those concerns should be separated: e.g. it should be possible for tests to use `CommonTestData` without using a mysql database. 

Simply moving `ClusterExtensions` to `TestComponent` is probably a good solution here.

Reply:
Hm, I think it'd be better for them to not depend on one another -- it's more modular that way. Ok to redefine `serviceAccountKey` if that's the only thing needed.